Sections/Set-Image-in-First-page-Header/.NET/Set-Image-in-First-page-Header/Set-Image-in-First-page-Header.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net8.0</TargetFramework>
-    <RootNamespace>Set_image_in_first_page_only</RootNamespace>
+    <RootNamespace>Set-Image-in-First-page-Header</RootNamespace>
```

Comment:
Modify casing of the sample name

Set_image_in_first_page_header

Reply:
Addressed